static/app/components/events/interfaces/spans/gapSpanDetails.tsx

```diff
@@ -41,11 +46,16 @@ export function GapSpanDetails({
 
   const profileId = event.contexts.profile?.profile_id || '';
 
-  // TODO: target the span's time window
-  const target = generateProfileFlamechartRoute({
+  // we want to try to go straight to the same config view as the preview
+  const query = canvasView?.configView
+    ? {fov: Rect.encode(canvasView.configView)}
```

Comment:
I think you want to force xAxis to profile or else this might be off by `profile.startedAt`